### PR TITLE
fixing some portuguese translations.

### PR DESCRIPTION
**Pull request by st693ava**

### CODE REVIEW

File: locales/pt/pt.json [hunk 1]
```diff
@@ -639,16 +639,16 @@
     "Tuvalu": "Tuvalu",
     "Two Factor Authentication": "Autenticação de dois fatores",
     "Two factor authentication is now enabled. Scan the following QR code using your phone's authenticator application.": "A autenticação de dois fatores está ativa. Digitalize o seguinte QR code utilizando a aplicação de autenticação do seu telefone.",
-    "Uganda": "Pai",
-    "Ukraine": "Ucrania",
+    "Uganda": "Uganda",
```
Comment (caouecs) [inline locales/pt/pt.json:642]: in exclude file

File: locales/pt/pt.json [hunk 2]
```diff
@@ -707,7 +707,7 @@
     "Your current payment method is a credit card ending in :lastFour that expires on :expiration.": "Your current payment method is a credit card ending in :lastFour that expires on :expiration.",
     "Your registered VAT Number is :vatNumber.": "Your registered VAT Number is :vatNumber.",
     "Zambia": "Zâmbia",
-    "Zimbabwe": "Pai",
-    "Zip \/ Postal Code": "Zip \/ Postal Code",
+    "Zimbabwe": "Zimbabwe",
```
Comment (caouecs) [inline locales/pt/pt.json:710]: in exclude file

File: locales/pt/validation-attributes.php [hunk 3]
```diff
@@ -1,40 +1,40 @@
...
-        'sex'                   => 'Sex',
-        'subject'               => 'Subject',
-        'terms'                 => 'Terms',
-        'time'                  => 'Time',
-        'title'                 => 'Title',
-        'username'              => 'Username',
-        'year'                  => 'Year',
-    ],
-];
+	'attributes' => [
```
Comment (caouecs) [inline locales/pt/validation-attributes.php:4]: Don't set space

File: locales/pt/validation-attributes.php [hunk 4]
```diff
@@ -1,40 +1,40 @@
...
+		'minute' => 'Minuto',
+		'mobile' => 'Telemóvel',
+		'month' => 'Mês',
+		'name' => 'Nome',
+		'password' => 'Palavra-passe',
+		'password_confirmation' => 'Confirmar Palavra-passe Confirma',
+		'phone' => 'Telefone',
+		'photo' => 'Fotografia',
+		'price' => 'Preço',
+		'role' => 'Role',
```
Comment (caouecs) [inline locales/pt/validation-attributes.php:30]: in exclude file

File: locales/pt/validation-inline.php [hunk 5]
```diff
@@ -12,122 +12,122 @@
...
-    'uploaded'             => 'Falhou ao carregar.',
-    'url'                  => 'Este formato é inválido.',
-    'uuid'                 => 'Este deve ser um UUID válido.',
-    'custom'               => [
-        'attribute-name' => [
-            'rule-name' => 'custom-message',
-        ],
-    ],
-];
+	'accepted' => 'Este campo deve ser aceite.',
```
Comment (caouecs) [inline locales/pt/validation-inline.php:15]: Don't set spaces

File: locales/pt/validation.php [hunk 6]
```diff
@@ -12,122 +12,122 @@
...
-    'uploaded'             => 'O upload do ficheiro :attribute falhou.',
-    'url'                  => 'O formato do URL indicado para o campo :attribute é inválido.',
-    'uuid'                 => ':attribute tem de ser um UUID válido.',
-    'custom'               => [
-        'attribute-name' => [
-            'rule-name' => 'custom-message',
-        ],
-    ],
-];
+	'accepted' => 'O campo :attribute deverá ser aceite.',
```
Comment (caouecs) [inline locales/pt/validation.php:15]: Don't set spaces

**Review by caouecs — CHANGES_REQUESTED** [review]

File: locales/pt/validation-attributes.php [hunk 7]
```diff
@@ -1,40 +1,40 @@
...
+		'second' => 'Segundo',
+		'sex' => 'Sexo',
+		'subject' => 'Assunto',
+		'terms' => 'Termos',
+		'time' => 'Hora',
+		'title' => 'Titulo',
+		'username' => 'Username',
+		'year' => 'Ano',
+	],
+];
```
Comment (olsza) [inline locales/pt/validation-attributes.php:40]: Format the style for readability as in the original...
```
        [...]
		'minute'                => 'Minuto',
		'mobile'                => 'Telemóvel',
		'month'                 => 'Mês',
		'name'                  => 'Nome',
		'password'              => 'Palavra-passe',
		'password_confirmation' => 'Confirmar Palavra-passe Confirma',
		'phone'                 => 'Telefone',
		'photo'                 => 'Fotografia',
		'price'                 => 'Preço',
        [...]
	],
];
```

Comment (olsza) [inline locales/pt/validation-attributes.php:40]: Give enter at the end of the file

File: locales/pt/pt.json (hunk 1 above)
Comment (olsza) [reply locales/pt/pt.json:642]: @st693ava in your case, add words to exclusions here "excludes/pt.php"

File: locales/pt/pt.json [hunk 8]
```diff
@@ -624,11 +624,11 @@
     "Togo": "Togo",
     "Tokelau": "Tokelau",
     "Token Name": "Nome do Token",
-    "Tonga": "Vir",
+    "Tonga": "Tonga",
```
Comment (andrey-helldar) [inline locales/pt/pt.json:627]: Specify this value in the exception file: `excludes/pt.php`

File: locales/pt/validation-attributes.php (hunk 3 above)
Comment (andrey-helldar) [reply locales/pt/validation-attributes.php:4]: No need to replace spaces with tabs

File: locales/pt/validation-attributes.php [hunk 9]
```diff
@@ -1,40 +1,40 @@
...
+		'photo' => 'Fotografia',
+		'price' => 'Preço',
+		'role' => 'Role',
+		'second' => 'Segundo',
+		'sex' => 'Sexo',
+		'subject' => 'Assunto',
+		'terms' => 'Termos',
+		'time' => 'Hora',
+		'title' => 'Titulo',
+		'username' => 'Username',
```
Comment (andrey-helldar) [inline locales/pt/validation-attributes.php:37]: Specify this value in the exception file: `excludes/pt.php`

**Review by andrey-helldar — CHANGES_REQUESTED** [review]
No need to replace spaces with tabs.

**Pull request closed without merging**